oauth2client/contrib/django_util/views.py

```diff
@@ -12,26 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""This module contains the views used by the OAuth2 flows.
+
+Their are two views used by the OAuth2 flow, the authorize and the callback
+view. The authorize view kicks off the three-legged OAuth flow, and the
+callback view validates the flow and if successful stores the credentials
+in the configured storage."""
+
 import hashlib
 import json
 import os
 import pickle
 
 from django import http
 from django import shortcuts
+from django.conf import settings
 from django.core import urlresolvers
+from django.shortcuts import redirect
+from six.moves.urllib import parse
 
 from oauth2client import client
 from oauth2client.contrib import django_util
+from oauth2client.contrib.django_util import get_storage
 from oauth2client.contrib.django_util import signals
-from oauth2client.contrib.django_util import storage
 
 _CSRF_KEY = 'google_oauth2_csrf_token'
 _FLOW_KEY = 'google_oauth2_flow_{0}'
 
 
 def _make_flow(request, scopes, return_url=None):
-    """Creates a Web Server Flow"""
+    """Creates a Web Server Flow
+
+    Args:
+        request: A Django request object
```

Comment:
Missing trailing punctuation on many, many parameter and return value descriptions.